packages/instrumentor/instrument.ts

```diff
@@ -98,6 +106,17 @@ export class Instrumentor {
 		return transformedCode;
 	}
 
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	private asOpts(inputSourceMap: any): any {
```

Comment:
It's used in `transform` and `sourceCodeCoverage`, and both take slightly different types. One expects the source map version to be a number and the other to be a string. I couldn't quite manage to use the concrete types in this function. 
Renamed it to `asInputSourceOption`, though.

Reply:
Do we have to convert it to raw source map before in line 88? Otherwise we could fix the type of the input:
```typescript
private asInputSourceOption(inputSourceMap: SourceMap | undefined): any {
		// Empty input source maps mess up the coverage report.
		if (inputSourceMap) {
			return {
				inputSourceMap,
			};
		}
		return {};
	}
```